### ISSUE TITLE
Spaces in data-i18n value for multiple parameters

### ISSUE THREAD

**Issue by jgobi** [i18next/jquery-i18next#34]
This could support to add a space after the ";" in the data-i18n attribute, such that 
```html
<input i18n-data="[value]nav.home; [placeholder]input.placeholder" />
```
will not result in a bug.

Currently, if this space is added, the i18next only understand the first "parameter". So, there is no option aside of don't using spaces, and this make the sintax for multiple "parameters" to be confusing to read.

**Comment by jamuhl** [comment]
PR welcome.

**Comment by jamuhl** [comment]
seems to work

published in jquery-i18next@1.2.1
